openreview/tools.py

```diff
@@ -359,6 +359,12 @@ def get_all_notes(client, invitation, limit=1000):
     '''
     return list(iterget(client.get_notes, invitation=invitation, limit=limit))
 
+def get_all_references(client, invitation, limit=1000):
```

Comment:
Why do you need to pass a limit to get all the references?

Reply:
May be the limit should be set in the get_all_references implementation. The user that tried to get all the references doesn't want to know about limit it just want to get all the references. It would be nice to return the iterator instead.

Reply:
Good catch! Mongo is failing because it is trying to sort without an index, I will take a look!